internal/app/storaged/server.go

```diff
@@ -40,6 +40,8 @@ func (s *Server) Disks(ctx context.Context, in *emptypb.Empty) (reply *storage.D
 			Name:       d.Name,
 			Serial:     d.Serial,
 			Modalias:   d.Modalias,
+			Uuid:       d.UUID,
```

Comment:
this won't fix it fully, the go-blockdevice also needs fixes, we'll take a look at fixing this for 1.7

Reply:
we might need a fix in `go-blockdevice`, but certainly this fix is needed as well!